chain/src/main/scala/org/bitcoins/chain/blockchain/ChainHandler.scala

```diff
@@ -384,7 +390,7 @@ case class ChainHandler(
       endOpt: Option[BlockStamp] = None,
       batchSize: Int = chainConfig.filterBatchSize,
       parallelismLevel: Int = Runtime.getRuntime.availableProcessors())(
-      implicit ec: ExecutionContext): Future[Vector[DoubleSha256DigestBE]] = {
+      ec: ExecutionContext): Future[Vector[DoubleSha256DigestBE]] = {
```

Comment:
why no implicit here if the `ec` still needs to be a parameter?

Reply:
The answer is in the scaladoc: this method performs better with a separate thread pool. But implicit `ec` clashes with the implicit `ExecutionContext` in the scope. This method is going to be moved to `wallet` soon anyway.

Reply:
The answer is in the scaladoc: this method performs better with a separate thread pool. But implicit `ec` clashes with the implicit `ExecutionContext` in the scope. This method is going to be moved to `wallet` soon anyway.

Reply:
The answer is in the scaladoc: this method performs better with a separate thread pool. But implicit `ec` clashes with the implicit `ExecutionContext` in the scope. This method is going to be moved to `wallet` soon anyway.